### PR TITLE
feat: sign in/up after installing extension

### PR DESCRIPTION
**Pull request by hstove**
Adds screens for the "after install path" of Connect 2.0

- After installing extension, automatically opens up to `/#/installed`
- Can sign up after install with one button
- Can sign in after install
- Integration tests for sign up / sign in after install

note: I have **not** implemented the password flow. That's going to take some refactoring of other parts of our code. I'd rather tackle that all separately.

### CODE REVIEW

File: packages/app/src/pages/secret-key.tsx [hunk 1]
```diff
@@ -65,6 +65,7 @@ export const SecretKey: React.FC<SecretKeyProps> = props => {
                 value={secretKey}
                 className="hidden-secret-key"
                 data-test="textarea-seed-phrase"
+                data-loaded={!!secretKey ? 'true' : 'false'}
```
Comment (kyranjamie) [inline packages/app/src/pages/secret-key.tsx]: Could just cast to a string `String(!!secretKey)`

File: packages/app/src/pages/install/sign-in.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  const textAreaRef = createRef<HTMLTextAreaElement>();
+  const [isLoading, setLoading] = useState(false);
+  const [seed, setSeed] = useState('');
+  const [seedError, setSeedError] = useState<null | string>(null);
+  const dispatch = useDispatch();
+  const { doChangeScreen, doTrack } = useAnalytics();
+
+  const hasLineReturn = (input: string) => input.includes('\n');
+
+  const onSubmit = async () => {
```
Comment (kyranjamie) [inline packages/app/src/pages/install/sign-in.tsx]: `useCallback()` ?

**Review by kyranjamie — APPROVED** [review]

File: packages/app/src/pages/install/sign-in.tsx (hunk 2 above)
Comment (hstove) [reply packages/app/src/pages/install/sign-in.tsx]: Yep - need to get into that habit.

**Comment by hstove** [comment]
@timstackblock this PR is ready for QA. It's only a small chunk of the Connect 2.0 work, but it'll be good to get started on QA as we move.

Expected behaviors are all listed in the description for this PR - note that the only difference from Figma is that we don't have the password flow yet. You can follow the first two rows of the figma to see what should be included here.

**Comment by timstackblock** [comment]
Thanks @hstove I will take a look

**Comment by markmhendrickson** [comment]
I tried testing this with the instructions provided by https://github.com/blockstack/ux/pull/670#issuecomment-724047589 but can't seem to trigger the `/#/installed` window either by selecting "Get your Secret Key" or "Install Extension" from the intro modal.

**Comment by hstove** [comment]
@markmhx , I haven't actually addressed the intro modal inside of the app, yet. The `/#/installed` page is automatically opened after you _install_ the extension. You can also QA this by simply going to [`/#/installed` in the hosted version](https://pr-670.app.stacks.engineering/#/installed).

**Pull request merged**